### DISCUSSION TITLE
cs.github.com bug: Incorrect language detection (rust vs. renderscript)

### DISCUSSION THREAD

**Discussion by trevyn** [community/community#18043] [Other Feature Feedback, Questions, & Ideas]
A bunch of examples at: https://cs.github.com/?scopeName=All+repos&scope=&q=rust+language%3ARenderScript

<img width="973" alt="Screen Shot 2022-06-07 at 11 31 57 AM" src="https://user-images.githubusercontent.com/230691/172422939-e09192ce-434e-40b7-bee3-2aa7cb19ead0.png">

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
